### DISCUSSION TITLE
Inertia.js adapter for v5

### DISCUSSION THREAD

**Discussion by eidellev** [adonisjs/core#2080] [Show and tell]
Hello Adonis community!

I've recently published an adapter for inertia.js:
https://www.npmjs.com/package/@eidellev/inertia-adonisjs

It would be great to get some feedback.
I'm planning on writing a short tutorial in the near future, but for now the README should help get you started.

**Comment by thetutlage** [comment]
Beautiful. I'll look into code tomorrow morning.

**Comment by dawidow** [comment]
@eidellev Amazing job. The AdonisJS community is amazing. Thank you for your work. I am already testing ;)

**Comment by ashokgelal** [comment]
This looks great!

**Comment by ammezie** [comment]
@eidellev nice!!!

**Comment by ANU8151** [comment]
I created boilerplate

https://github.com/ANU8151/InertiaJS-Vue3-Tailwind-CSS-AdonisJS

**Reply by eidellev** [reply]
@ANU8151 This really awesome. Thanks!
